diff -durN bzflag.orig/configs/config-linux bzflag/configs/config-linux
--- bzflag.orig/configs/config-linux	Thu Mar 15 14:31:03 2001
+++ bzflag/configs/config-linux	Thu Mar 15 14:35:39 2001
@@ -64,8 +64,8 @@
 #
 PCOPTS   = -Wall -W -ansi $(ACOPTS)
 PCXXOPTS = -Wall -W -fno-exceptions -fno-rtti $(ACXXOPTS)
-COPT     = -DNDEBUG -O2
-CXXOPT   = -DNDEBUG -O2 -ffast-math -fomit-frame-pointer -fnonnull-objects \
+COPT     = $(COPTFLAGS) -DNDEBUG
+CXXOPT   = $(COPTFLAGS) -DNDEBUG -ffast-math -fomit-frame-pointer -fnonnull-objects \
 		-fexpensive-optimizations
 CDEBUG   = -DDEBUG -g
 CXXDEBUG = -DDEBUG -g
