diff -durN bzflag.orig/Make-common bzflag/Make-common
--- bzflag.orig/Make-common	Tue Jan 11 05:00:15 2000
+++ bzflag/Make-common	Sun Apr  9 14:11:35 2000
@@ -28,7 +28,7 @@
 #
 # default install directories (configs can override these)
 #
-INSTALL_DATA_DIR = /usr/local/lib/bzflag
+INSTALL_DATA_DIR = /usr/X11R6/share/bzflag
 
 #
 # C options
diff -durN bzflag.orig/configs/config-linux bzflag/configs/config-linux
--- bzflag.orig/configs/config-linux	Sat Jan 15 19:39:35 2000
+++ bzflag/configs/config-linux	Sun Apr  9 14:10:28 2000
@@ -23,7 +23,7 @@
 #
 # install locations
 #
-INSTALL_DATA_DIR = /usr/X11R6/lib/X11/bzflag/data
+INSTALL_DATA_DIR = /usr/X11R6/share/bzflag
 
 #
 # tools
