diff -durN bzflag.orig/src/bzfs/bzfs.cxx bzflag/src/bzfs/bzfs.cxx
--- bzflag.orig/src/bzfs/bzfs.cxx	Fri Mar  9 22:10:33 2001
+++ bzflag/src/bzfs/bzfs.cxx	Thu Mar 15 14:39:37 2001
@@ -15,7 +15,7 @@
 // to enforce a game time limit
 #define TIMELIMIT
 // to dump score info to stdout
-//#define PRINTSCORE to include code to dump score info to stdout
+#define PRINTSCORE to include code to dump score info to stdout
 
 
 // Like verbose debug messages?
