--- bzflag-1.10.6.20040515/configure.in.orig	2004-05-15 01:09:04.000000000 +0200
+++ bzflag-1.10.6.20040515/configure.in	2004-10-17 00:32:41.714841988 +0200
@@ -125,8 +125,8 @@
 
 
 # for BeOS - old network stack don't have those libs ( move it in the case switch ?)
-AC_CHECK_LIB(socket, socket)
-AC_CHECK_LIB(bind, gethostent)
+AC_SEARCH_LIBS(socket, socket)
+AC_SEARCH_LIBS(gethostent, bind)
 
 AC_CHECK_LIB(m, sqrtf)
 
